src/scheduler.py

```
@@ -164,131 +209,165 @@ async def _save_result_and_update_picks(session, match, winner, score_str):
+        await session.commit()
+    logger.info("Saved result and updated picks for match %s.", match.id)
+
+    await send_result_notification(match, result)
+
+    if job_id is None:
+        job_id = f"poll_match_{match.id}"
+    logger.info("Unscheduling job %s for completed match.", job_id)
+    _remove_job_if_exists(job_id)
+
+
+async def _should_continue_polling(session, match: Match | None, job_id: str) -> bool:
```

Ruff (0.14.10)

Unused function argument: `session`

(ARG001)